core/src/main/java/org/apache/iceberg/TableMetadata.java

```diff
@@ -74,11 +74,7 @@ public static TableMetadata newTableMetadata(
   public static TableMetadata newTableMetadata(
       Schema schema, PartitionSpec spec, String location, Map<String, String> properties) {
     SortOrder sortOrder = SortOrder.unsorted();
-    int formatVersion =
-        PropertyUtil.propertyAsInt(
-            properties, TableProperties.FORMAT_VERSION, DEFAULT_TABLE_FORMAT_VERSION);
-    return newTableMetadata(
-        schema, spec, sortOrder, location, persistedProperties(properties), formatVersion);
+    return newTableMetadata(schema, spec, sortOrder, location, persistedProperties(properties));
```

Comment:
No need to call persistedProperties here, it has already been invoked internally

Reply:
you are right!